frontend/components/top_nav/SiteTopNav/_styles.scss

```diff
@@ -95,7 +96,10 @@
 
 .site-nav-list {
   list-style: none;
+  width: 671px;
```

Comment:
why the set width here? is this needed?

Reply:
I either saw that number somewhere in the style guide, the "masthead styles," or found it to preserve the width of each nav item with some new layout choices.